website/content/v1.5/talos-guides/install/virtualized-platforms/proxmox.md

```diff
@@ -105,6 +105,10 @@ Finish creating the VM by clicking through the "Confirm" tab and then "Finish".
 Repeat this process for a second VM to use as a worker node.
 You can also repeat this for additional nodes desired.
 
+> Note: Talos OS doesn't support hot plugging, if creating the VM programmatically don't enable memory hotplug on your
```

Comment:
```suggestion
> Note: Talos doesn't support memory hot plugging, if creating the VM programmatically don't enable memory hotplug on your
```

Reply:
Thanks for the suggested changes I will get onto that tomorrow. Out of curiosity, is it just memory hotplug that is not supported or is all hot plugging not supported such as CPU, Disk and USB?

Reply:
blockdevice/PCI hotplug should definitely work. not sure if Kubernetes supports memory hotplugging (?)